editor.planx.uk/src/routes/team.tsx

```diff
@@ -76,4 +96,17 @@ const routes = compose(
   }),
 );
 
+const fetchDraftFlattenedFlowData = async (
```

Comment:
@jessicamcinchak here's the thing I added and mentioned in our Slack chat*


* *in the Slack chat, we discussed issues with switching from flowName and flowSlug state changes to condensing into a setFlow state change, but the change led to issues with reading ("_root" and "edges") which got resolved by fetching the flowData*